### ISSUE TITLE
Usage with gql function from codegen (client-preset)

### ISSUE THREAD

**Issue by chrisregner** [apollographql/vscode-graphql#95]
Instead of apollo's `gql` function, I use a [generated `gql` for TypeScript support](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client) like this

```
import { gql } from '__generated__'


const MY_QUERY = gql(/* GraphQL */ `
  ...myQuery
`)
```

Is it possible to integrate this extension to support this?

**Comment by duongdev** [comment]
+1 I need this too

**Comment by scottjr632** [comment]
I'm not sure if this is officially supported, but I found a workaround for the time being where you can add `#graphql` next to the beginning tick. 

_Example_
```typescript
import { graphql } from "@/__generated__/gql";
const VIEWER_QUERY = graphql(`#graphql
  query PageQuery {
    viewer {
      login
    }
  }
`)
```

They seem to have support this with an `|` check [here in the regex](https://github.com/apollographql/vscode-graphql/blob/1b91ab4f2ac06f4b4e25ae7aeec8f767aa989759/src/language-server/document.ts#L93-L96) that gets created to match templates.

**Comment by Foresteam** [comment]
Another workaround: update apollo config with this:
```js
module.exports = {
	client: {
		tagName: '/', // /👍👍👍👍/
		...
	}
};
```
And now you can use it like this (works with any strings, though):
```js
gql(/* GraphQL */`
   query {...}
`);

**Comment by kyldvs** [comment]
https://github.com/apollographql/vscode-graphql/pull/99 fixes this if you install the latest update of the extension

**Comment by alessbell** [comment]
Yes, thanks @kyldvs - I'll close this out as resolved in #99, for any other feature requests/bugs please open a new issue :)
